src/services/__tests__/helpers.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('helpers', () => {
       mocked(getExerciseProgress).mockReturnValueOnce(Promise.resolve({ '10': { '0': 1, '1': 1 } }))
       const { disciplineId, exerciseKey } = await getNextExerciseWithCheck()
       expect(disciplineId).toBe(mockDisciplines()[0].leafDisciplines![0])
-      expect(exerciseKey).toBe(2)
+      expect(exerciseKey).toBe(1)
```

Comment:
The title of this test is "should open third exercise of first discipline, if two exercise were finished yet", but it returned 1, not as before 2. This is because the two exercieses, that are finished, are not finished good enough, there score is only 1, as you can see in Line 29. So if you want to fix this test correctly, you need to change back the expected result to 2, and the score of the done exercises above the threshold.